restart-relay.sh

```diff
@@ -37,7 +37,7 @@ blocktime=${T=0}
 pkill -f ganache-cli && echo killed old ganache.
 pkill -f RelayHttpServer && echo kill old relayserver
 
-GANACHE="$root/node_modules/.bin/ganache-cli -l 8000000 -b $blocktime -a 11 -h 0.0.0.0 "
+GANACHE="ganache-cli -l 8000000 -b $blocktime -a 11 -h 0.0.0.0 "
```

Comment:
use "`npx ganache-cli`, to use local ganache, global.

Reply:
yes i am changing it to `npx ganache-cli` and `npx truffle` as requested in the issue